src/components/BaseKey.vue

```diff
@@ -34,67 +34,62 @@ let substitute = Object.assign(
   colorways.platformIcons(window.navigator.platform)
 );
 
-const _getUnitClass = (unith, unitw) => {
-  if (unith == unitw && unith > 1) {
-    return 'custom';
+const _getKeyClass = (unith, unitw) => {
+  if (unith === 2 && unitw == 1.25) {
```

Comment:
move this test below the unith=1 as it's getting checked every time and is the least common branch

Reply:
actually move this above the custom